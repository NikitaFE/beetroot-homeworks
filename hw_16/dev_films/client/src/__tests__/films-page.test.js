import React from 'react';
import {render, cleanup, shallow} from '@testing-library/react';
import {MemoryRouter as Router} from 'react-router-dom';
import FilmsPage from '../components/FilmsPage';

describe('<FilmsPage />', () => {
  afterEach(cleanup);

  it('should make snapshot', () => {
    const {container} = render(<Router><FilmsPage location={{pathname: '/'}} /></Router>);
    expect(container.firstChild).toMatchSnapshot();
  });
});


