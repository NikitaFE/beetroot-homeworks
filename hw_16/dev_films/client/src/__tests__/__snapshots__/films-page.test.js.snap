// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilmsPage /> should make snapshot 1`] = `
<div
  class="ui stackable grid"
>
  <div
    class="ten wide column"
  >
    <div
      class="ui four cards"
    >
      <div
        class="ui icon olive message"
      >
        <i
          class="icon star outline"
        />
        <div
          class="content"
        >
          <div
            class="header"
          >
            No films in out base yet
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
